app/scripts/config/index.js

```diff
@@ -10,7 +10,7 @@ const baseConfig = {
   environment: 'development',
   requireEarthdataLogin: false,
   apiRoot: 'https://wjdkfyb6t6.execute-api.us-east-1.amazonaws.com/dev/',
-  compatibleApiVersions: ['1.11.0'],
+  minCompatibleApiVersions: ['1.11.0'],
```

Comment:
The way things were written, a "minimum compatible version" would only match on minor patches. i.e. `1.11.0` would include any `1.11.x`, but not `1.12.0`.

I just re-wrote the functionality so that `1.11.0` will match with `1.(11+x).y` for all x and y >= 0

Reply:
This feels simpler and less prone to mis-interpretation. I am only a little worried about new api versions not being backward-compatible, but I think this is a good solution for now.